openmetadata-spec/src/main/resources/json/schema/entity/applications/configuration/private/limits.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/applications/configuration/private/appLimitsConfig.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "AppLimitsConfig",
+  "description": "Private Configuration for the App Limits.",
+  "type": "object",
+  "javaType": "io.collate.schema.entity.app.Limits",
+  "definitions": {
+    "action": {
+      "type": "object",
+      "javaType": "io.collate.schema.entity.app.LimitAction",
+      "properties": {
+        "name": {
```

Comment:
since actions should be unique, it might make more sense to use a `Map<String,Integer>`. This way developers cannot mistakenly define duplicate actions.

Reply:
applying this in Collate, thanks, makes sense